partiql-spi/src/main/kotlin/org/partiql/value/io/PartiQLValueIonReader.kt

```diff
@@ -124,15 +122,7 @@ internal class PartiQLValueIonReader(
                 }
             }
 
-            IonType.SYMBOL -> {
-                if (reader.isNullValue) {
-                    symbolValue(null, reader.typeAnnotations.toList())
-                } else {
-                    symbolValue(reader.stringValue(), reader.typeAnnotations.toList())
-                }
-            }
-
-            IonType.STRING -> {
+            IonType.STRING, IonType.SYMBOL -> {
```

Comment:
For the `PartiQLValueIonReader` changes, what's the intended behavior for Ion types that we are not representing in PartiQL? Seems like this PR coerces `SYMBOL` to PartiQL string values and `SEXP`s to PartiQL list values. Should we give an error instead?